Integrations docs (#2808)
mattermost/docs: source/deployment/productivity-tools.rst

Productivity Tools
============================================

Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect productivity tools in Mattermost.

Here are some popular options below, including self-hosted on-prem or self-hosted private cloud solutions, and vendor-hosted SaaS solutions.

Matterpoll for polls
~~~~~~~~~~~~~~~~~~~~~~~~

 - Create a poll by using a slash command, developed by Hanzei and Yusuke Nemoto.

>  - Polls can be created in different supported languages, currently French, German and Japanese.

Reviewer: ```suggestion
 - Polls can be created in different languages, including English, French, German and Japanese.
```

 - Source code + docs: https://github.com/matterpoll/matterpoll

Google Calendar
~~~~~~~~~~~~~~~~~~~~~~~~

>  - Time management tool, developed by Wasim Thabraze.

Reviewer: ```suggestion
 - Time management tool for Google Calendar, developed by Wasim Thabraze.
```

>  - Get reminders about appointments and meetings from configured Google Calendar to any Mattermost channel.

Reviewer: ```suggestion
 - Get reminders about appointments and meetings from a configured Google Calendar to any Mattermost channel.
```

 - Source code + docs: https://github.com/waseem18/mattermost-plugin-google-calendar

CloudApp
~~~~~~~~~~~~~~~~~~~~~~~~

 - Tool for screen recording, screenshots, and GIF creation in the cloud, developed by CloudApp team.
 - Record a screen, generate a GIF or create a screenshot, and share in Mattermost.
 - Docs: https://www.getcloudapp.com/integrations/mattermost

Gyazo
~~~~~~~~~~~~~~~~~~~~~~~~

>  - Screenshot app, developed by Sean Sackowitz.

Reviewer: ```suggestion
 - Screenshot application, developed by Sean Sackowitz.
```

 - Take screenshots, screen GIFs and full page captures, and share in Mattermost.
 - Source code: https://github.com/seansackowitz/mattermost-plugin-gyazo

Autotranslation Plugin
~~~~~~~~~~~~~~~~~~~~~~~~

>  - Two-way integration between Amazon AWS Translate and Mattermost, developed by Mattermost.

Reviewer: This may be a one-way integration, not sure

Reviewer 2: changed the text to be ambiguous
